infra/bootstrap_utils.sh

```diff
@@ -7,29 +7,6 @@ fi
 
 source $HAIL/devbin/functions.sh
 
-render_config_mk() {
-    DOCKER_PREFIX=$(get_global_config_field docker_prefix)
-    INTERNAL_IP=$(get_global_config_field internal_ip)
-    IP=$(get_global_config_field ip)
-    DOMAIN=$(get_global_config_field domain)
-    CLOUD=$(get_global_config_field cloud)
```

Comment:
Are SCOPE and DEPLOY not needed anymore?

Reply:
They're still in config.mk, but they don't vary by cloud so there's no reason to re-render the file.

Reply:
I forgot to include those in the PR message, they haven't changed.

Reply:
My bad. I see now.